docs/source/commands.rst

```diff
@@ -119,6 +119,25 @@ optional arguments:
 
     $ cfncluster delete mycluster
 
+login
+======
+
+Attempts to ssh into head node. Stack needs to be in CREATE_COMPLETE or UPDATE_COMPLETE.
+Head node needs to be running. You can specify an ssh `key_path` yo use in your config or as a cli parameter.
+If you specify it as a cli parameter, this will override the value in the config.
+
+positional arguments:
+  cluster_name  ssh into head node
+
+optional arguments:
+  -h, --help        show this help message and exit
+  -s, --key-path    path to ssh private key

```

Comment:
The shorthand variant of the argument should start with the same character as that of the full argument. Either go for `-k`, or follow the same convention as the ssh command and take the identity file with `-i`.  I'd clarify the help text too, and explicitly say key file instead of just path.  Path would typically refer to the path that contains the file, so `~/.ssh/` instead of ~/.ssh/my_priv_key_file`.

Reply:
Implementing `ssh` flags is a slippery slope. Ideally there's a way for the customer to pass arbitrary flags and commands that'll just get run.

Reply:
I'm not saying we should implement the ssh flags, that is definitely a slippery slope. My comment was merely about the naming of this login command's argument to take a key.